lib/cadet_web/router.ex

```diff
@@ -125,6 +116,11 @@ defmodule CadetWeb.Router do
   scope "/v2", CadetWeb do
     pipe_through([:api, :auth, :ensure_auth])
 
+    get("/assessments", AssessmentsController, :index)
+    post("/assessments/:assessmentid", AssessmentsController, :show)
```

Comment:
This ought to become a `GET` (I think I mentioned this before); are you going to do this later on?

Reply:
I wasn't sure about how to do the following:

- Split `POST /assessments/{assessmentId}` into two separate requests to unlock assessments with passwords
  - I'm not sure how we might want to do this for now: maybe add a password column to the submissions table, so that `POST /assessments/{assessmentId}/unlock` would add the user-input password into that column, and `GET /assessments/{assessmentId}` can compare passwords between the Assessment and Submission?

Reply:
The existing logic is that it creates an empty submission the first time the assessment is opened with the password. Subsequently, once a submission exists, no password is required.

So you could just move the accept password &rarr; create empty submission into a new endpoint.